server/pse/pse_windows.go

```diff
@@ -255,7 +256,12 @@ func ProcUsage(pcpu *float64, rss, vss *int64) error {
 		return fmt.Errorf("could not find pid in performance counter results")
 	}
 	// assign values from the performance counters
-	*pcpu = cpuAry[idx]
+	cpu := cpuAry[idx]
+	if cpu > 0.0 {
+		cpu /= float64(runtime.NumCPU())
```

Comment:
I wonder if you should capture the number of CPUs in the init function. I guess this could change dynamically, but...
Also, we do that in monitor.go